launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml

```diff
@@ -15,6 +15,8 @@
       threshold_time_object_is_moving: 1.0        # [s]
       object_check_forward_distance: 150.0        # [m]
       object_check_backward_distance: 2.0         # [m]
+      object_check_shiftable_ratio: 0.6           # [-]
+      object_check_min_road_shoulder_width: 0.5   # [m]
```

Comment:
I think these three related parameters should be rearranged, add white space on top and bottom so that it is simpler to be found

1. threshold_distance_object_is_on_center
2. shiftable ration
3. min road shoulder width

Currently threshold_distance_object_is_on_center is on top of the list.

Reply:
Thanks for your advise :smile: I'll fix it.

Reply:
I fixed in https://github.com/autowarefoundation/autoware.universe/pull/2329/commits/a94873dc7027ed548ed4636045cc844b6b462077